src/crypto/index.js

```diff
@@ -246,8 +246,12 @@ Crypto.prototype.uploadDeviceKeys = function() {
     });
 };
 
+Crypto.prototype.updateCurrentKeyCount = function(currentKeyCount) {
+     _maybeUploadOneTimeKeys(this, currentKeyCount);
+};
+
 // check if it's time to upload one-time keys, and do so if so.
-function _maybeUploadOneTimeKeys(crypto) {
+function _maybeUploadOneTimeKeys(crypto, currentKeyCount) {
```

Comment:
can you document the currentKeyCount parameter? what is the behaviour when it is passed, or not passed?